src/css/main.less

```diff
@@ -472,20 +473,22 @@ fieldset[disabled] .btn-info.active {
 }
 
 .nav-tabs > li > a {
-  color: #8e929b;
+  color: @navbar-inactive-color;
   border-top: none;
   border-left: none;
   border-right: none;
   border-radius: 0;
+  min-width: @navbar-min-width;
```

Comment:
I think what we should do here is derive the `min-width` from a value that is more related to the font size of the element than anything else.